common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -246,6 +248,29 @@ def publish_completion(self, data, dispatch):  # pylint: disable=unused-argument
         self.runtime.publish(self, "completion", data)
         return {"result": "ok"}
 
+    @staticmethod
+    def make_transcript_http_response(content, filename, language, content_type):
+        """
+        Construct `Response` object.
+
+        Arguments:
+            content (unicode): transcript content
+            filename (unicode): transcript filename
+            language (unicode): transcript language
+            mimetype (unicode): transcript content type
+        """
+        response = Response(
+            content,
+            headerlist=[
+                ('Content-Disposition', 'attachment; filename="{}"'.format(filename.encode('utf-8'))),
```

Comment:
We don't need transcript as an attachment in case of `/translations/{lang}` dispatch.